pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryTestSet.java

```diff
@@ -26,6 +26,11 @@ public class QueryTestSet {
   @DataProvider(name = "testSql")
   public Object[][] provideTestSql() {
     return new Object[][]{
+        // Failed left join. Not sure what's the expectation here.
+        // new Object[]{"SELECT a.col1 FROM a LEFT JOIN b ON (a.col1 = 'foo')"},
```

Comment:
also FYI this is a cross join (since there's no rule for b)

Reply:
Removed.